src/winetricks

```diff
@@ -2637,13 +2646,15 @@ w_metadata()
 
     # Do sanity check unless running on Cygwin, where it's way too slow.
     case "$W_PLATFORM" in
-        windows_cmd)
-            ;;
-        *)
+        windows_cygwin) w_warn "FIXME: Sanity check for w_metadata() is too slow, skipping.." ;;
+        windows_windows) w_die "FIXME: Platform $W_PLATFORM is not implemtented in w_metadata()" ;;
```

Comment:
Not needed.

Reply:
> windows_windows) w_die "FIXME: Platform $W_PLATFORM is not implemtented in w_metadata()" ;;

same reasoning as in https://github.com/Winetricks/winetricks/pull/1338/files/7c0eed4e8a25108c4f9e345657356343a07cbf78#r332377677